integration/tests/basic/rpc/test_rpc_get_transaction.py

```diff
@@ -193,7 +217,9 @@ def test_eth_get_transaction_by_hash(self, json_rpc_client):
             ["blockHash", "blockNumber", "hash", "transactionIndex", "type", "from", "to"],
             {"hash": "transactionHash"},
         )
+        EthGetTransactionByHashResult(**response)
 
+    # Geth does not have effectiveGasPrice in response
```

Comment:
Is the issue created?

Reply:
On a new version of Geth it does not reproduce anymore. Removing the comment.